src/test/java/org/gridsuite/shortcircuit/server/repositories/FaultResultRepositoryTest.java

```diff
@@ -131,6 +141,38 @@ void faultResultFilterWithPageableTest(ShortCircuitAnalysisResultEntity resultEn
         assertFaultEqualsInOrder(faultPage, Comparator.comparing(FaultResultEntity::getNbLimitViolations).thenComparing(o -> o.getFaultResultUuid().toString()));
     }
 
+    @ParameterizedTest(name = "[{index}] Using the filter(s) {1} should return the given entities")
+    @MethodSource({
+        "provideFeederFieldsStartFiltersSortedAsc",
+        "provideFeederFieldsContainFiltersSortedDesc",
+        "provideFeederFieldsUnfilteredSortedAscAndDesc"
+    })
+    void feedersFilterTest(ShortCircuitAnalysisResultEntity resultEntity,
+                           List<ResourceFilter> resourceFilters,
+                           Sort.Order sort,
+                           List<List<FeederResult>> expectedFeedersLists) {
+        Page<FaultResultEntity> faultPage = shortCircuitAnalysisResultRepository.findFaultResultsPage(
+                resultEntity,
+                resourceFilters,
+                PageRequest.of(0, 3, Sort.by(
+                        new Sort.Order(Sort.Direction.ASC, "current"), sort)),
+                FaultResultsMode.FULL);
+
+        List<List<String>> feedersConnectableIds = faultPage.getContent().stream()
+                        .map(faultRes -> faultRes.getFeederResults().stream()
+                                .map(FeederResultEntity::getConnectableId)
+                                .toList()
+                        ).toList();
+
+        List<List<String>> expectedFeedersConnectableIds = expectedFeedersLists.stream()
+                        .map(feederList -> feederList.stream()
+                                .map(FeederResult::getConnectableId)
+                                .toList()
+                        ).toList();
+
+        Assertions.assertEquals(expectedFeedersConnectableIds, feedersConnectableIds);
```

Comment:
I'm not sure using assertEquals on the list is the best way to compare them. AssertJ has many methods to do that specifically.

Reply:
I did it with `assertThat(feedersConnectableIds).isEqualTo(expectedFeedersConnectableIds);`

Is is ok ?

Reply:
I think it's the same, but nevermind, we compare String so it's not a problem anyway

Reply:
It's just that instead of creating 2 new List to compare the IDs, you could have use `extracting` and `containsExactlyElementsOf` as above